backend/inspirehep/matcher/parsers.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from inspire_dojson.utils import strip_empty_values
+from inspire_schemas.api import ReferenceBuilder
+from parsel import Selector
+
+
+class GrobidParser:
```

Comment:
```suggestion
class GrobidReferenceParser:
```
Maybe also document that this parses a single reference (with root node `biblStruct`).